internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer.go

```diff
@@ -91,6 +91,9 @@ func (*Sequencer) Initialize(r runtime.Runtime) []runtime.Phase {
 		)
 	default:
 		phases = phases.Append(
+			"mountTmpfs",
+			MountTmpFilesystems,
```

Comment:
I know you did that coming from a perspective of doing things consistently, but I think we should drop this from the sequencer.

The problem is that controllers and sequencer start at the same time, and it's very inconvenient not to have `/tmp` or `/system` at this point.

So following #8897, I would rather move it out to be a proper function which wraps the initialization as a whole.

This addresses also your concern about unmounting... so something like:

```go
func initEarly() err {
   unmounter, err := mountTmpFilesystems()
   if err != nil { ... }
   
  defer unmounter()
  
  doOtherStuff()
}
```

(and we need to skip this for containers, so place this accordingly)

Reply:
Well, my primary concern for review here is whether or not it's okay and not too late time to mount these? It worked for me, but I'm not quite sure initramfs init won't need either of these.